CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Add `CsrRenderedInitials`
 * Support `CsrRenderedInitials` mesh morphing
 * Setup `CsrRenderedInitials`
+* Add `loadFBX` and `loadEnvironment` methods to utils
```

Comment:
There is no global or general issue for RIPE CSR?

Reply:
Nope

Reply:
Now you do: https://github.com/ripe-tech/ripe-sdk/issues/429